andela/micro-py-health-check
# #146764007 Set up Python micro-health-check package

#### What does this PR do?
Creates a python package to do rpc Health checks
#### Description of Task to be completed?
Create a python package that will be used to do health checks on all Python micro-services
#### How should this be manually tested?
Set up a python `virtualenv` and then run `pip install https://github.com/andela/micro-py-health-check/archive/ft-python-healthcheck-pypi-package-146764007.zip`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#146764007
